server/src/main/java/io/spine/server/delivery/DeliveryBuilder.java

```diff
@@ -221,6 +225,9 @@ public DeliveryBuilder setInboxStorage(InboxStorage inboxStorage) {
      *
      * <p>If none set, the storage is initialized by the {@code StorageFactory} specific for
      * this {@code ServerEnvironment}.
+     *
+     * <p>If no {@code StorageFactory} is present in the {@code ServerEnvironment}, a new
+     * {@code InMemoryStorage} is used.
```

Comment:
There is no such thing as `InMemoryStorage`. Please review your changeset for this copy-paste mistake.